sdk/daml-lf/ledger-api-value/src/main/protobuf/com/daml/ledger/api/v2/value.proto

```diff
@@ -116,21 +116,13 @@ message RecordField {
 
 // Unique identifier of an entity.
 message Identifier {
-  // Both package_id and package_name is provided as reading from the Ledger API
-  // If provided in Ledger API requests the following rules apply:
-  //
-  // - If package_name is provided and package_id is provided referring to a package name (``#`` notation): name based package selection is preferred, and the defined package name must match.
-  // - If package_name is provided and package_id is provided referring to a package ID: ID based package selection is preferred, and the package name of the package with the package_id should match the package_name.
-  // - If package_name is provided and package_id is not provided: name based package selection is preferred.
-  // - If package_name is not provided and package_id is provided referring to a package name (``#`` notation): name based package selection is preferred.
-  // - If package_name is not provided and package_id is provided referring to a package ID: ID based package selection is preferred.
-  // - If package_name is not provided and package_id is not provided: this is illegal.
-
   // The identifier of the Daml package that contains the entity.
   // Must be a valid PackageIdString.
-  // This is always provided as reading from the Ledger API.
-  // This is not required as provided on Ledger API commands/requests.
-  // Optional
+  //
```

Comment:
@nmarton-da : I did the rewording for two reasons:
1. to understand what you wrote and play back to you whether I understood it correctly, did I?
2. to move the emphasis away from the deprecated behavior

Could you check whether the wording I propose matches your intution? and seperately from that decide what you want to adopt from it?

thx :)